README.md

```diff
@@ -176,6 +176,21 @@ If you got any problems with SPF and/or forwarding mails, give [SRS](https://git
 
 See the [documentation](https://docker-mailserver.github.io/docker-mailserver/edge/config/security/understanding-the-ports/) for further details and best practice advice, **especially regarding security concerns**.
 
+#### Folders
+
+By default, `INBOX`, `Drafts`, `Sent`, `Junk`, `Trash` are created and automatically subscribed. Please take a look at [`target/dovecot/15-mailboxes.conf`][gh-config-dovecot-mailboxes] for existing folder definitions as reference. A useful example is to enable the `Archive` folder, allowing different mail clients to share the same archive folder automatically. 
```

Comment:
Same here

Reply:
I think folder is quite standard when referring to IMAP folders? At least when I was still working as sysadmin few years ago?

Reply:
Standard as in user / admin choice of word, or official terminology? "Folder" is not the technical term in the RFC correct?

It's fine to mention it as a commonly known name to those that are more used to the user perspective of it. eg: `Mailboxes (_aka Mail Folders_)`

Reply:
Mailbox should cause confusions. You can just google "IMAP folders" and "IMAP Mailbox". I think "folder" is a common and widely used wording in both user / admin.

Reply:
> Mailbox should cause confusions.

We're referencing an RFC about the feature you're contributing documentation for, there is no mention of "folder" or "tag". Just because a non-technical user may not be familiar with the mailbox term does not make it invalid or causing confusion.

We also link to official Dovecot wiki about what you're documenting, and config files that explicitly have mailbox in the name and file content. So no, this is not a poor choice to prefer.

I understand that some users may be more familiar with a "folder" concept for it instead, and that may be the first thing that comes to mind when performing a search query to learn how to configure such, that adds some bias, especially since they're more likely to use a keyword like "folder" instead of "mailbox" because they presumably don't know the correct term.

I don't mind you using "folder" or "tag" in the documentation, but there should be a very clear indication of the official terms. You can do this as advised by using official terminology like "Mailbox" in the headings and page name (title and filename + mkdocs entry), or "Mailbox folder" for example which covers both keywords.

The headings can provide an alternative user-friendly/familiar name like has been suggested between parentheses: `Mailbox (aka Folder)` or similar. Then the body content can use whatever word you think is better for the reader, so long as the communication is clear.

Reply:
Mailbox usually refers to the whole IMAP structure. And folder refers to individual folders..
There is no official term. If you might not know, RFC does not necessarily mean "official" or "standard".

Reply:
My problem was not the wording of "folder" but what "shared" means by this topic:

> Does this really mean "share" the same folder (between accounts) or "share" the same configuration?

Reply:
> Mailbox usually refers to the whole IMAP structure. And folder refers to individual folders..

Like I said, this isn't something I'm personally familiar with. But between the RFC, the referenced Dovecot docs on the subject and the related config files as examples... The term "Mailbox"/"Mailboxes" is rather evident when referring to this.

- Microsoft seems to refer to them as [IMAP folders](https://support.microsoft.com/en-us/office/choose-which-imap-account-folders-appear-in-outlook-32103602-a568-4499-9d25-54fe236aee6a). Other parts of the docs reference "user mailboxes".
- Apple Mail refers to them as [mailboxes](https://support.apple.com/en-nz/guide/mail/mlhlp1021/mac). They even have the term [mailbox](https://support.apple.com/en-nz/guide/mail/aside/glos8e4eb14f/14.0/mac/11.0) defined with description in their support document.
- [Google references the RFC](https://developers.google.com/gmail/imap/imap-extensions#special-use_extension_of_the_list_command) Special-Use Mailboxes noting it's relevance to "special folders", this is developer documentation though. Looking through consumer docs suggest that it has no concept of mailboxes/folders, the user only manages via labels.. I believe some other clients call this [Virtual Folders](https://en.wikipedia.org/wiki/Virtual_folder#Email_clients).
- Courier seems to [only have a single INBOX mailbox](https://www.cyrusimap.org/imap/concepts/features/automatic-creation-of-mailboxes.html), and describes everything else as sub-folders belonging to it. [`imapd.conf` has a `specialuse_extra` field](https://www.cyrusimap.org/imap/reference/manpages/configs/imapd.conf.html) noting the attributes are per mailbox. A [Personal Namespace](https://www.cyrusimap.org/imap/concepts/features/namespaces.html#imap-features-namespaces-personal) depicts the expected default hierarchy though while mixing mailboxes and sub-folders of a mailbox in the description.

[Wikipedia further communicates](https://en.wikipedia.org/wiki/Internet_Message_Access_Protocol#Multiple_mailboxes_on_the_server) what I was trying to get across:

> _IMAP4 clients can create, rename, and/or delete mailboxes (usually presented to the user as folders) on the server, and copy messages between mailboxes._

LCN has a support article on [IMAP mailbox folders](https://www.lcn.com/support/articles/syncing-your-mailbox-folders-using-imap/), it ments "sent", "drafts", "trash" as standard set of folders for your mailbox. Then lists instructions for various clients, where you'll see different jargon like covered above:

- Windows Live Mail interestingly refers to these "standard folders" as "Special folders on IMAP server".
- Thunderbird has dedicated UI for Sent, Archives, Drafts and Templates and presumably Junk.

---

Having glanced over these resources I link you to, and the [RFC 6154](https://datatracker.ietf.org/doc/html/rfc6154.html#page-3) document again, it's rather clear that a mailbox can be commonly known as representing the Archive, Sent, Trash and Junk "folders".

> There is no official term. If you might not know, RFC does not necessarily mean "official" or "standard".

RFC documents are often what is referenced for implementing protocols that are standardized for interoperability between various software no? IMAP is one of those AFAIK. That does not mandate software use the same terminology in their service or GUI, but it's pretty official on the functional / technical level.

> _IMAP4rev1 permits manipulation of mailboxes (remote message folders) in a way that is functionally equivalent to local folders. - [Source (RFC 3501)](https://datatracker.ietf.org/doc/html/rfc3501)_

Quick look through the document is fairly clear with the mailbox term and it's description. There's also the [LIST command](https://datatracker.ietf.org/doc/html/rfc3501#section-6.3.8) (RFC 3501 - 6.3.8) also further supports that. There's also [section 5.1 on "Mailbox Naming"](https://datatracker.ietf.org/doc/html/rfc3501#section-5.1):

> _The case-insensitive mailbox name INBOX is a special name reserved to mean "the primary mailbox for this user on this server". The interpretation of all other names is implementation-dependent._

---

Do you disagree that the ambiguity to a user or mail client UI warrants being a little more specific than "folders", when that term is not clearly defined in the docs page?

All I've asked is to make it clear what "folder" relates to early on in the document or title.

Reply:
> My problem was not the wording of "folder" but what "shared" means by this topic

@wernerfred from the little reading I have done above and reviewing of this contribution, "shared" is probably not the right word to use.

I am under the impression the various mail clients are intended to implicitly recognize the "Archives" IMAP folder / mailbox and know it's purpose (from the special use attribute assigned to the mailbox).